### DISCUSSION TITLE
wxhelper-3.9.0.28-specify-port-V13 注入之后，无法卸载，请问如何解决.

### DISCUSSION THREAD

**Discussion by caiyakang** [ttttupup/wxhelper#257] [General]
成功注入之后，
使用D:\project\mtp\hook\wxhelper-3.9.0.28-specify-port-V13\ConsoleInject.exe -u WeChat.exe -d wxhelper.dll 进行卸载，发现里面的dll还注存在里面
![image](https://github.com/ttttupup/wxhelper/assets/12150755/8ea26275-9116-45b8-a160-6e316ce2f8ad)
请问有什么办法可以完全卸载，主要是取消http的端口监听。

卸载失败很大可能原因是因为这个dll启动了HTTP监听，那怎么样可能停止这个HTTP监听了？

![image](https://github.com/ttttupup/wxhelper/assets/12150755/8cadf061-d11d-4854-a189-df1afe8f9629)

@ttttupup 请问我如何调用到里面这个http_close

![image](https://github.com/ttttupup/wxhelper/assets/12150755/f57aa875-983e-46d4-9f52-6e05647b69e7)

**Comment by ttttupup** [comment]
源码里怎么调用http_start 的就怎么调用http_close

**Comment by ttttupup** [comment]
注入工具源码里不都有吗

**Comment by caiyakang** [comment]
目前源码里面没有调用http_close，我还不会C++，暂时改不动。

**Comment by ttttupup** [comment]
注入进去了，直接可以在dll调用。没注入的话用远程线程注入调用。
